packages/@aws-cdk/cdk/test/test.app.ts

```diff
@@ -296,7 +296,25 @@ export = {
 
     delete process.env.JSII_AGENT;
     test.done();
-  }
+  },
+
+  'version reporting includes only @aws-cdk, aws-cdk and jsii libraries'(test: Test) {
```

Comment:
I feel this test isn't very string. It operates on an extremely anecdotical example where there is not much room for surprise (almost a self-fulfilling prophecy).

Would at least also test the filter function with inputs we can directly control as part of the test.

Reply:
It's going to be too hard to actually unit test this in isolation, and now that I've changed the logic to basically only include "@aws-cdk/*" modules, I think it's not worth it. We know that that the runtime environment of the test includes other modules, so this test will fail if the filter fails.

Reply:
Makes sense! The new filter is a lot stricter, so I'm wayyyyy less concerned now (particularly since we *own* the `@aws-cdk` scope).